lib/components/ArtistScreen/artist_item_info.dart

```diff
@@ -0,0 +1,100 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:get_it/get_it.dart';
+
+import '../../models/jellyfin_models.dart';
+import '../../screens/artist_screen.dart';
+import '../../services/jellyfin_api_helper.dart';
+import '../../services/process_artist.dart';
+import '../print_duration.dart';
+
+class ArtistItemInfo extends StatelessWidget {
+  const ArtistItemInfo({
+    Key? key,
+    required this.item,
+    required this.itemSongs,
+    required this.itemAlbums,
+  }) : super(key: key);
+
+  final BaseItemDto item;
+  final int itemSongs;
+  final int itemAlbums;
+
+// TODO: see if there's a way to expand this column to the row that it's in
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+      children: [
+        _IconAndText(
+            iconData: Icons.music_note,
+            text: AppLocalizations.of(context)!.songCount(itemSongs)),
+        _IconAndText(
+            iconData: Icons.book,
+            text: AppLocalizations.of(context)!.albumCount(itemAlbums))
```

Comment:
The album count isn't correct for me for some artists. It should probably use `albumCount` instead of `childCount`, but that's a more expensive operation and probably shouldn't be used for all artists, that's why it's currently not being requested from the server for the artists tab items. You could try requesting it for all artists, or use a `FutureBuilder` and only show the album info after you re-fetched the artists when a user opens the ArtistScreen...

I quickly tried to also request the `AlbumCount` field from the server, but for some reason that didn't work. Maybe you can give it another try? 🤔

Reply:
I'll address this in the other PR and linked this comment there, this slid into this one on accident

Reply:
I have no idea how that happened lol 👀